package.json

```diff
@@ -55,6 +55,8 @@
     "react-dom": "^15.3.0 || ^16.0.0-beta.1"
   },
   "dependencies": {
+    "@types/enzyme": "^2.8.4",
```

Comment:
Problem is that the MUI typings depend on them and won't work if they're not present. If you're using TypeScript it will not compile.

Maybe move them to `optionalDependencies`? But I guess most users will have `react` and `enzyme` types installed anyway?